components/users/usersService.js

```diff
@@ -99,7 +99,8 @@ class usersService {
       this.collection,
       new UserModel({ ...user, password: hashedPassword, username: username })
     );
-    return createUserId, username;
+
+    return { createUserId, username };
```

Comment:
Does it not affect another function?

Reply:
Answer is not, I checked service createUser